tests/lax_numpy_ufuncs_test.py

```diff
@@ -322,6 +412,33 @@ def test_reduceat(self, func, nin, nout, identity, shape, axis, idx_shape, dtype
     self._CheckAgainstNumpy(jnp_fun, np_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @jtu.sample_product(
+      JAX_NUMPY_UFUNCS_WITH_DTYPES,
+      [{'shape': shape, 'axis': axis}
+       for shape in nonscalar_shapes
+       for axis in [*range(-len(shape), len(shape))]],
+      idx_shape=[(0,), (3,), (5,)],
+  )
+  def test_ufunc_reduceat(self, name, shape, axis, idx_shape, dtype):
+    jnp_fun = getattr(jnp, name)
+    np_fun = getattr(np, name)
+    if (jnp_fun.nin, jnp_fun.nout) != (2, 1):
+      self.skipTest(f"accumulate requires (nin, nout)=(2, 1); got {(jnp_fun.nin, jnp_fun.nout)=}")
```

Comment:
Does this skip often? If so, maybe sample from a filtered version of `JAX_NUMPY_UFUNCS_WITH_DTYPES`? If not then this is fine to keep.

Reply:
Currently it never skips, but it might someday

Reply:
I'll probably revisit this as I expand the number of defined ufuncs